packages/wallet-api-manifest-validator/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@ledgerhq/wallet-api-manifest-validator",
+  "version": "1.0.0",
+  "description": "This package checks if your manifest.json file meets the requirements for Ledger Live App manifest submission.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsup src/ && tsup cli",
+    "build:src": "tsup src",
+    "format:check": "prettier --check \"src\" \"tests\"",
+    "format:fix": "prettier --write \"src\" \"tests\"",
+    "lint": "eslint --cache --ext .ts \"src\" \"tests\"",
+    "lint:fix": "eslint --cache --fix --ext .ts \"src\" \"tests\"",
+    "prepare": "tsup src/ && tsup cli",
+    "test": "jest tests/*.test.*",
```

Jest doesn't need the file globing here and I don't think it works with the param actually
```suggestion
    "test": "jest",
```